### PR TITLE
Add vineyard+mars+pytorch as a showcase in vineyard operator blog

### PR DESCRIPTION
**Pull request by dashanji**
What do these changes do?
-------------------------

1. As titled.
2. Fix https://github.com/v6d-io/v6d/issues/1162

### CODE REVIEW

File: docs/notes/how-to-use-vineyard-operator.rst [hunk 1]
```diff
@@ -624,7 +624,242 @@ above object as follows.
...
+    etcd1                                          1/1     Running   0          68s
+    etcd2                                          1/1     Running   0          68s
+    vineyard-controller-manager-7f569b57c5-46tgq   2/2     Running   0          92s
+    vineyardd-sample-6ffcb96cbc-gs2v9              1/1     Running   0          67s
+    vineyardd-sample-6ffcb96cbc-n59gg              1/1     Running   0          67s
+    vineyardd-sample-6ffcb96cbc-xwpzd              1/1     Running   0          67s
+
+.. raw:: html
+
+   </details>
```
Comment (sighingnow) [inline docs/notes/how-to-use-vineyard-operator.rst:803]: Using `raw` html in both rst and markdown is quite hacky. We would try to implement the `.. details::` markup in upstream theme.

Comment (dashanji) [reply docs/notes/how-to-use-vineyard-operator.rst:803]: Emmm, for `markdown`, there is a label like `<details>`. Do you mean to add a markup to [the repo](https://github.com/pradyunsg/furo)?

Comment (sighingnow) [reply docs/notes/how-to-use-vineyard-operator.rst:803]: Will implement inside vineyard itself.

**Comment by sighingnow** [comment]
Closing as manually merged in https://github.com/v6d-io/v6d/pull/1253/commits/68d12eb7d1359a63a84db2bbb88744e713891faa

**Pull request closed without merging**